eng/pipelines/runtime-official.yml

```diff
@@ -56,7 +56,6 @@ stages:
       - Linux_musl_arm64
       - Windows_NT_x86
       - Windows_NT_x64
-      - Windows_NT_arm
```

Comment:
This is the official build, which means if we remove this, we will stop producing an `arm` runtime. Is that desired?

Reply:
This is desired. However we may want to be the last people to remove so the upstack components do not break.